src/GenerateCommand.php

```diff
@@ -185,4 +190,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln("Verify what I have done and do <info>git push</info> to deploy the application "
             . "or <info>git reset --hard origin/master</info> to rollback all changes.");
     }
+
+    private function getGithubToken(InputInterface $input, OutputInterface $output): string
+    {
+        // Use CLI option if present
+        $cliToken = $input->getOption('github-token');
+        if ($cliToken) {
+            $output->writeln("Using GitHub API token from the command line.");
+            return $cliToken;
+        }
+
+        // Request token from user
+        $output->writeln("Please provide a GitHub token from the https://github.com/settings/tokens");
+        $output->writeln("For required scopes, see https://docs.travis-ci.com/user/github-oauth-scopes");
+
+        $helper = $this->getHelper('question');
+        $question = new Question('GitHub token:');
+        $githubToken = $helper->ask($input, $output, $question);
+
+        // Remove previous line and replace token with *****
+        if ($output->isDecorated()) {
```

Comment:
Idealne by bolo namiesto skopirovaneho tokenu zobrazovat `***`, no to v Symfony console nejde. Da sa iba uplne skryt vstup, ... no to je pri kopirovani blbe, nevies ci si token skopiroval cely, ci si ho nevlozil 2x, ... kedze po vlozeni sa nic nestane. ..... preto sa token po vlozeni zobrazi, ale po stlaceni ENTER sa nahradi `***` ak to terminal podporuje.

Reply:
Zmazanie riadku je prebrate z Symfony Console ProgressBar: https://github.com/symfony/console/blob/3179d1350843d2caa796973173dd0609ed4c7c67/Helper/ProgressBar.php#L386